plugins/org.python.pydev.parser/src/org/python/pydev/parser/jython/ast/Python.asdl

```diff
@@ -108,6 +111,9 @@ module Python
 	     | Name(identifier id, expr_context ctx, bool reserved)
 	     | List(expr* elts, expr_context ctx)
 	     | Tuple(expr *elts, expr_context ctx, bool? endsWithComma)
+	     
+	     -- added in version 3.10
```

Comment:
I think that the changes to the .asdl should be the same that python did (see the related diff at: https://github.com/python/cpython/commit/145bf269df3530176f6ebeab1324890ef7070bf8#diff-56cfb21eb86b07bb94e916eb74143c7d6c2ac6f2494216c0e2507fd080c34aed).